pages.es/linux/zypper.md

```diff
@@ -0,0 +1,28 @@
+# zypper
+
+> Utilidad para la gestión de paquetes en SUSE y openSUSE.
+> Más información: <https://en.opensuse.org/SDB:Zypper_manual>
+
+- Actualiza todos los repositorios:
+
+`zypper refresh`
+
+- Instala un nuevo paquete:
+
+`zypper install {{paquete}}`
+
+- Elimina un paquete:
+
+`zypper remove {{paquete}}`
+
+- Actualiza los paquetes instalados con versiones nuevas:
+
+`zypper update`
+
+- Busca en los repositorios un paquete mediante una palabra clave:
+
+`zypper search {{palabra_clave}}`
+
+- Muestra un listado con la información de los repositorios configurados:
+
+`zypper repos -P`
```

Comment:
This example is not in the main (English) page.

Reply:
yes, I know. I though that would be ok to add a example about how zypper also can manage repos

Reply:
Would it be possible that you could add this example to the English page also?